packages/compass-indexes/src/components/indexes-toolbar/indexes-toolbar.tsx

```diff
@@ -28,6 +30,10 @@ const toolbarButtonsContainer = css({
   alignItems: 'center',
 });
 
+const alignSelfEnd = css({
```

Comment:
Similar to other styles defined in this file, we usually use `Styles` at the end of the variable name for styles

Reply:
OK, fixed in 21d152e.